modules/openapi-generator/src/main/resources/typescript-inversify/api.service.mustache

```diff
@@ -73,21 +73,21 @@ export class {{classname}} {
         if ({{paramName}}) {
         {{#isCollectionFormatMulti}}
             {{paramName}}.forEach((element) => {
-                queryParameters.push("{{paramName}}="+encodeURIComponent(String({{paramName}})));
+                queryParameters.push('{{paramName}}='+encodeURIComponent(String({{paramName}})));
             })
         {{/isCollectionFormatMulti}}
         {{^isCollectionFormatMulti}}
-            queryParameters.push("{{paramName}}="+encodeURIComponent({{paramName}}.join(COLLECTION_FORMATS['{{collectionFormat}}'])));
+            queryParameters.push('{{paramName}}='+encodeURIComponent({{paramName}}.join(COLLECTION_FORMATS['{{collectionFormat}}'])));
         {{/isCollectionFormatMulti}}
         }
         {{/isListContainer}}
         {{^isListContainer}}
         if ({{paramName}} !== undefined) {
         {{#isDateTime}}
-           queryParameters.push("{{paramName}}="+encodeURIComponent(<any>{{paramName}}.toISOString()));
+           queryParameters.push('{{paramName}}='+encodeURIComponent(({{paramName}} as Date).toISOString()));
```

Comment:
what is the original type of `{{paramName}}`? can it be casted to Date without issues?

Reply:
The type of paramName normally implements Date interface so this change should be correctly.

Reply:
I think the general type for `format: date-time` in the typescript generators is `string`, see https://github.com/OpenAPITools/openapi-generator/pull/5314

Reply:
@macjohnny 
> I think the general type for `format: date-time` in the typescript generators is `string`, see #5314

I'm pretty sure that is Date interface here, because I'm using it this generator.